mailbox/api/src/test/java/org/apache/james/mailbox/extractor/ContentTypeFilteringTextExtractorTest.java

```diff
@@ -0,0 +1,71 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.extractor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.io.InputStream;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.google.common.collect.ImmutableList;
+
+public class ContentTypeFilteringTextExtractorTest {
+
+    @Mock
+    TextExtractor textExtractor;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void extractContentReturnEmptyWithContentTypeInBlacklist() throws Exception {
+        ContentTypeFilteringTextExtractor contentTypeFilteringTextExtractor =
+            new ContentTypeFilteringTextExtractor(textExtractor,
+                TextExtractorConfiguration.builder()
+                    .contentTypeBlacklist(ImmutableList.of("application/ics", "application/zip"))
+                    .build());
+
+        assertThat(contentTypeFilteringTextExtractor
+            .extractContent(null, "application/ics"))
+            .isEqualTo(ParsedContent.empty());
```

Comment:
Verify there was no interactiction with the backend text extractor 

`Mockito.verifynoMoreInteractions(...);`